components/canvas_traits/lib.rs

```diff
@@ -4,6 +4,7 @@
 
 #![crate_name = "canvas_traits"]
 #![crate_type = "rlib"]
+#![feature(nonzero)]
```

Comment:
It's not clear to me why this feature is required?

Reply:
It’s probably not. This looks left over from before a rebase.